### PR TITLE
[WEB-610] Sentry configuration

### PR DESCRIPTION
**Pull request by sydorovdmytro**
Related to https://github.com/Livit/Livit.Learn.LTI/pull/174
Also added OverviewXBlock as requirement

Before deploy just add to server_vars.yml

```
EDXAPP_AUTH_EXTRA:
  RAVEN_CONFIG:
   dsn: "< put here DSN from Sentry >"
```

@polesye @TamoshaytisV please review

### CODE REVIEW

File: cms/envs/labster.py [hunk 1]
```diff
@@ -19,3 +19,8 @@
 )
 
 LABSTER_WIKI_LINK = LABSTER_SETTINGS.get('LABSTER_WIKI_LINK', 'https://theory.labster.com/')
+
+# Sentry integration config
+RAVEN_CONFIG = AUTH_TOKENS.get('RAVEN_CONFIG', {})
+if RAVEN_CONFIG['dsn']:
```
Comment (polesye) [inline cms/envs/labster.py]: you will get KeyError exception if `RAVEN_CONFIG` is not set. Use `RAVEN_CONFIG.get('dsn')`

File: lms/envs/labster.py [hunk 2]
```diff
@@ -36,3 +36,8 @@
     LABSTER_ENDPOINTS[endpoint] = value
 
 LABSTER_DEFAULT_LTI_ID = LABSTER_SETTINGS.get('LABSTER_DEFAULT_LTI_ID', 'MC')
+
+# Sentry integration config
+RAVEN_CONFIG = AUTH_TOKENS.get('RAVEN_CONFIG', {})
+if RAVEN_CONFIG['dsn']:
```
Comment (polesye) [inline lms/envs/labster.py]: the same here

File: cms/envs/labster.py (hunk 1 above)
Comment (sydorovdmytro) [inline cms/envs/labster.py]: Fixed

File: lms/envs/labster.py (hunk 2 above)
Comment (sydorovdmytro) [inline lms/envs/labster.py]: Fixed

**Comment by sydorovdmytro** [comment]
@avkoval @madyasiwi please review if you can spare a moment

**Comment by polesye** [comment]
👍

**Comment by TamoshaytisV** [comment]
👍

**Pull request merged**